spec/models/auth/voms_spec.rb

```diff
@@ -28,6 +28,30 @@
     end
   end
 
+  describe '#parse_hash_groups!' do
+    context 'with all kinds of groups' do
+      let(:env_hash) do
+        { 'GRST_VOMS_FQANS' => '/coolclub/Role=NULL/Capability=NULL;' \
+                               '/nerds/Role=NULL/Capability=NULL;' \
```

Comment:
This is probably something we haven't talked about at all because I forgot to mention it ... sorry. In VOMS, groups can have subgroups. This would look like `/coolclub/muchcoolersubclub/Role=NULL/Capability=NULL`.

Now, we DO NOT want to support subgroups, let me be clear on that. But we should have a clearly defined behavior when such credentials arrive.

I would suggest just dropping subgroups. That would mean dropping every matched group with `/` in its name (as matched by `%r{^\/(?<group>[^\s]+)\/Role=(?<role>[^\s]+)\/Capability=NULL$}`). Opinions?

Reply:
I agree.